src/tkn.ts

```diff
@@ -628,9 +635,9 @@ export class TknImpl implements Tkn {
 
     async _getPipelineRuns(pipeline: TektonNode): Promise<TektonNode[]> | undefined {
         const result = await this.execute(Command.listPipelineRuns(pipeline.getName()));
-        if (result.stderr) {
+        if (getStderrString(result.error)) {
```

Comment:
I think it is ambiguous  to call `getStderrString ` there, it better to check that `result.error` is not `undefined`. So it will be better to replace with `if (result.error) {`